codewell-fe/src/Components/User/Login.js

```diff
@@ -1,12 +1,25 @@
 import React, { useState } from 'react';
 
+import { useLocation } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { Alert, Button, Card, Form, Input, Layout, Space } from 'antd';
 import { UserOutlined, LockOutlined } from '@ant-design/icons';
 
+import { getInitialAlertDescription } from '../../Helpers/Login.js';
+
 function Login(props) {
   const { t } = useTranslation();
-  const [error, setError] = useState('');
+  const location = useLocation();
+
+  /**
+   * To display a message in case of successful registration, the 'state'
+   * property of the 'location' object is inspected
+   */
+  const initialAlertDescription = getInitialAlertDescription(location.state);
+
+  const [alertDescription, setAlertDescription]
+    = useState(initialAlertDescription);
+  const [alertType, setAlertType] = useState('success');
```

Comment:
let's also say that types can alert have, something like this:
```suggestion
  /**
   * Alert type can be: success/error/whatever
   */
  const [alertType, setAlertType] = useState('success');
```

Reply:
Should I list them all? success, info, warning, error